elements/elrepo/pre-install.d/01-elrepo

```diff
@@ -8,20 +8,15 @@ set -euo pipefail
 
 case "$DISTRO_NAME" in
     centos* | rhel* | rocky*)
-	dnf install -y elrepo-release.noarch
```

Comment:
Old indentation made more sense to me

Reply:
That might be the case. I'll just drop it then.